packages/wonder-blocks-clickable/src/components/clickable-behavior.js

```diff
@@ -88,6 +88,8 @@ type CommonProps = {|
 
     skipClientNav?: boolean,
 
+    tabIndex?: number,
```

Comment:
**suggestion:** It would be helpful to add a comment explaining this prop here. I see that you added it in `argtypes.js`, but it's also helpful to add it in the implementation context.

Side note: I hope we can get rid of these comments duplication in the future 🤞